akka-http-backend/src/main/scala/com/softwaremill/sttp/akkahttp/AkkaHttpBackend.scala

```diff
@@ -303,24 +330,22 @@ class AkkaHttpBackend private (
 }
 
 object AkkaHttpBackend {
-  private def apply(
+  private def make(
```

Comment:
The compiler was complaining about overloads with similar shapes...

Reply:
yeah, happens :)